### PR TITLE
Add `dim-bots` feature

### PR DESCRIPTION
**Pull request by fregante**
Closes https://github.com/sindresorhus/refined-github/issues/1676 

# Test

https://github.com/typed-ember/ember-cli-typescript/commits/master
https://github.com/OctoLinker/OctoLinker/commits/master
https://github.com/OctoLinker/OctoLinker/pulls?q=is%3Apr+is%3Aclosed (this list may still be painful to your eyes)

commits

prs

### CODE REVIEW

**Comment by fregante** [comment]
prs

File: source/features/dim-bots.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+In PR lists we can hide the meta just by acting on `max-height` + `overflow: hidden`
+In Commit lists however this is not possible because they are tables, so we need to hack around `margin-bottom`
+
+Transitions are used to delay the "hover" status so it's not too annoying when moving the mouse over a list of dimmed items.
+*/
+.rgh-dim-bot.commit .commit-title,
+.rgh-dim-bot.commit .commit-meta,
+.rgh-dim-bot.commit .commit-links-cell summary,
+.rgh-dim-bot.commit .commit-links-cell .btn {
+	transition: 0.1s 0.3s;
```
Comment (fregante) [inline source/features/dim-bots.css]: Titles are dimmed and meta info/buttons are hidden. This effect is undone with a slight delay on hover/focus, currently set to `0.3s`

File: source/features/dim-bots.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	transition-property: opacity, margin-bottom;
+}
+
+.rgh-dim-bot.commit:not(.navigation-focus) .commit-title {
+	opacity: 0.2;
```
Comment (fregante) [inline source/features/dim-bots.css]: Is this value too much?

Comment (kidonng) [reply source/features/dim-bots.css]: Maybe `0.3` like we did to footer?

**Comment by sindresorhus** [comment]
> Should this feature be disabled for repo owners?

👍

**Comment by FloEdelmann** [comment]
I don't think this feature should be disabled for repo owners. When I look at a commit list or closed PR list in my own repo, I want to focus on the real changes, just like a visitor (or contributor) does.

The only place where it maybe would make sense to disable would be the open PR list, but also there I'd opt for dimming.

**Comment by sindresorhus** [comment]
> I don't think this feature should be disabled for repo owners. When I look at a commit list or closed PR list in my own repo, I want to focus on the real changes, just like a visitor (or contributor) does.

So don't use spammy bots in the first place?

**Comment by FloEdelmann** [comment]
When I don't need spammy bots, I don't need the feature to be disabled in my own repo :wink:

**Comment by fregante** [comment]
That’s irrelevant, when there are no bots, the feature doesn’t apply. The argument is whether to show something to the people who enabled *something.* 

For every optional feature we hide, we always keep showing it to the repo owner, because if they’re don’t want to see it, then they should disable it. The same concept applies here.

**Pull request merged**